pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -333,8 +341,10 @@ message PodSandboxStatus {
     optional LinuxPodSandboxStatus linux = 6;
     // Labels are key value pairs that may be used to scope and select individual resources.
     map<string, string> labels = 7;
-    // Annotations is an unstructured key value map that may be set by external
-    // tools to store and retrieve arbitrary metadata.
+    // Unstructured key value map that may be set by external tools to store
+    // and retrieve arbitrary metadata.
+    // The value of this field MUST be identical to that of the corresponding
+    // PodSandboxConfig used to instantiate the pod sandbox this status represents.
     map<string, string> annotations = 8;
```

Comment:
> If that's the case, it seems like the annotations on the PodSandbox and PodSandboxStatus messages are redundant?

Spoke to @timstclair offline. The podsandbox annotations today include the annotations from the k8s pod (similar to labels). I forgot about that completely. 
We can add in the comment that kubelet pass the pod annotations to the podsandbox.

I think we can change the comment to reflect that.